main.py

```diff
@@ -1,10 +1,40 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players = json.load(f)
+    for player in players:
+        guild = players[player].get("guild")
+        guild_instance = None
+        if guild is not None:
+            guild_instance = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"]
+            )[0]
+        race = Race.objects.get_or_create(
+            name=players[player]["race"]["name"],
+            description=players[player]["race"]["description"]
+        )[0]
+        skills = players[player]["race"]["skills"]
+        for skill in skills:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+        player_info = players[player]
+        Player.objects.get_or_create(
+            nickname=player,
+            email=player_info["email"],
+            bio=player_info["bio"],
+            race=race,
+            guild=guild_instance
+            if guild is not None else None
```

Comment:
It is strange. Your code loops like:
```
if guild_instance is not None:
   guild  = guild_instance
else:
    guild = guild_instance
```
```suggestion
            guild=guild_instance
```